source/index.html

```diff
@@ -36,18 +35,20 @@ <h2>Cool Things About Me!</h2>
           <p>Food truck seitan normcore plaid, meh 8-bit prism hashtag vape. Vice typewriter celiac chillwave brunch, fixie 8-bit listicle microdosing. Ramps snackwave mustache edison bulb PBR&B prism photo booth. Listicle venmo normcore enamel pin 90's, semiotics brooklyn gentrify kombucha kickstarter.</p>
-          <p>Deep v sustainable cardigan austin, put a bird on it bitters offal fingerstache master cleanse. Meggings semiotics green juice, 8-bit banjo trust fund forage selvage cold-pressed gluten-free authentic microdosing pickled. Hashtag banh mi blue bottle, chicharrones chartreuse kitsch forage microdosing vape.</p> 
+          <p>Deep v sustainable cardigan austin, put a bird on it bitters offal fingerstache master cleanse. Meggings semiotics green juice, 8-bit banjo trust fund forage selvage cold-pressed gluten-free authentic microdosing pickled. Hashtag banh mi blue bottle, chicharrones chartreuse kitsch forage microdosing vape.</p>
           <p>Sustainable pok pok ennui forage bespoke subway tile. Microdosing cray pitchfork, schlitz woke vegan salvia irony sustainable locavore. Raw denim meggings ugh brooklyn chia, blog YOLO iceland master cleanse cardigan coloring book authentic thundercats craft beer. Kombucha poke shoreditch, vaporware freegan yuccie small batch gluten-free migas meditation chia pok pok pabst ennui offal.</p>
         </section>
         <section>
           <h3>Experiences</h3>
-          <p>Kinfolk vinyl deep v, shoreditch single-origin coffee ramps flannel brooklyn tote bag street art pok pok. Aesthetic iPhone 8-bit, +1 migas street art next level brooklyn fanny pack williamsburg gluten-free. Drinking vinegar offal seitan stumptown chartreuse artisan enamel pin flannel, gluten-free prism hell of 8-bit retro gentrify pork belly.</p> 
+          <p>Kinfolk vinyl deep v, shoreditch single-origin coffee ramps flannel brooklyn tote bag street art pok pok. Aesthetic iPhone 8-bit, +1 migas street art next level brooklyn fanny pack williamsburg gluten-free. Drinking vinegar offal seitan stumptown chartreuse artisan enamel pin flannel, gluten-free prism hell of 8-bit retro gentrify pork belly.</p>
           <p>Sustainable sartorial vinyl craft beer distillery. Poke lomo air plant, edison bulb pinterest vice listicle 3 wolf moon. Slow-carb lo-fi DIY, 8-bit coloring book semiotics chicharrones green juice snackwave blue bottle next level. +1 succulents cronut cliche, banjo XOXO intelligentsia hot chicken mumblecore williamsburg wayfarers church-key flexitarian 8-bit.</p>
         </section>
       </div>
       <div class="footer">
         Hope you enjoyed this!
       </div>
     </div>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <script src="styleEditor.js"></script>
```

Comment:
I understand why you moved these to the bottom of the HTML so that they were loaded after the page load. Could we have done something different in our script to make sure eventListeners only ran after the page loaded? maybe `$(document).ready(function(){ })`